source/appModules/notepad++.py

```diff
@@ -0,0 +1,16 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2022 NV Access Limited, Łukasz Golonka
+# This file may be used under the terms of the GNU General Public License, version 2 or later.
+# For more details see: https://www.gnu.org/licenses/gpl-2.0.html
+
+"""This is just an alias which maps the appModule for Notepad++ to the right binary file
+by exposing everything from the real module in its namespace.
+"""
+
+import nvdaBuiltin.appModules.notepadPlusPlus as baseAppMod

```

Comment:
I'm puzzled by the complexity of this:

1. Why should core import from nvdaBuiltin?
2. Why not just import * from the module?

Reply:
This is partially a  remnant from the old approach where the dummy nodule was named notepadPlusPlus and the main one was notepad++. This has been necessary since you cannot perform relative import with `importlib` star imports cannot be performed neither. I can change this if you want me to, but given that this would hopefully be in master only for a few days I don't think this is necessary.

Reply:
Let's keep the pattern for appModule aliases consistent for now if possible:

```suggestion
from .notepadPlusPlus import *
```

Reply:
This has been done.